src/test/java/com/hotels/jasvorno/JasvornoConverterTest.java

```diff
@@ -359,4 +665,136 @@ public void unionOneAmbiguousMatchPromotedToFull() throws Exception {
     assertThat(unionResolution.schema, is(a)); // expected
   }
 
+  @Test
+  public void unionDoubleInFloatRangeMax() throws JsonProcessingException, IOException {
+    Schema a = SchemaBuilder.builder().floatType();
```

Comment:
We try avoid single character variable names (with the exception of an index in a loop) so lets call this schemaA and the other one schemaB (same for other tests below).